liblwgeom/lwout_encoded_polyline.c

```diff
@@ -91,7 +91,7 @@ char * pointarray_to_encoded_polyline(const POINTARRAY *pa, int precision)
 	for (i=0; i<pa->npoints*2; i++)
 	{
 		/* Left-shift the binary value one bit */
```

Comment:
it's left-shift in comment and *2 in code, can this be synced?

Reply:
I'll edit it to clarify it a bit